src/Models/CompanyRole.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace TruckersMP\Models;
+
+use Carbon\Carbon;
+
+class CompanyRole
+{
+    /**
+     * The ID of the requested role.
+     *
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * The name of the role.
+     *
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * The order in which the role is displayed.
+     *
+     * @var int
+     */
+    protected $order;
+
+    /**
+     * The owner of the role.
```

Comment:
This comment does not seem to be correct. I suggest these changes:

```suggestion
     * The owner of the company.
```